content/research_report/methods.md

```diff
@@ -16,13 +16,13 @@ kernelspec:
 ## Datasets
 
 We selected two datasets on OpenNeuro for the current analysis:
-`ds000228` {cite:p}`ds000228:1.1.0` and `ds000030` {cite:p}`ds000030:1.0.0`.
-Dataset `ds000228` (N = 155) contains fMRI scans of participants watching a silent version of Pixar animated movie "Partly Cloudy".
+`ds000228` {cite:p}`ds000228:1.1.0` and `ds000030` {cite:p}`ds000030:1.0.0`. <!-- NC: Maybe remove this from the intro since it's just been stated there, and probably belongs in the methods instead?  -->
```

Comment:
Sounds good! There's no harm having it there. I will do this when we merge the intro.